.commitlintrc.yml

```diff
@@ -0,0 +1,29 @@
+rules:
+  description-empty:
+    level: error
+  subject-empty:
+    level: error
+  body-empty:
+    level: error
+  scope:
+    level: error
+    optional: false
+    options:
+      - adapters
+      - object-model
+      - db-controller
+      - db-builder
```

Comment:
we use dbs to abbreviate in other parts of the code base so might be nice to keep that consistent